config/initializers/two_factor_auth_controller_override.rb

```diff
@@ -18,6 +18,10 @@ def handle_two_factor_authentication
         end
       end
 
+      def teaspoon_controller?
```

Comment:
although, I believe doing this will prevent a dev from being able to run the specs in the browser?